utils/deploy_netchecker.sh

```diff
@@ -10,7 +10,7 @@ fi
 kubectl get nodes
 
 echo "Installing netchecker server"
-echo '
+cat << EOF > {{ kube_config_dir }}/netchecker-server-pod.yml
```

Comment:
oops, I forgot to remove these jinja2 vars while copy pasted from ansible templates...